integration/testdata/gke_loadbalancer-render/skaffold.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: skaffold/v3alpha1
+kind: Config
+build:
+  artifacts:
+    - image: gke-loadbalancer
+      context: ./
+manifests:
+  helm:
+    releases:
+    # seed test namespace in the release name.
```

Comment:
is this actually done here?  In the other yaml there is a namespace var:
 - name: skaffold-helm-{{.TEST_NS}}
 
 Just want to make sure the comment makes sense here

Reply:
nope! This is just a leftover comment from cloning another example. Will remove